dataactcore/utils/report.py

```diff
@@ -9,18 +9,32 @@ def get_report_path(job, report_type):
     Return the filename for the error report.
     Does not include the folder to avoid conflicting with the S3 getSignedUrl method.
     """
-    path = 'submission_{}_{}_{}_report.csv'.format(job.submission_id, job.file_type.name, report_type)
-    return path
+    return report_file_name(job.submission_id, report_type == 'warning',
```

Comment:
Can do -- I didn't want to worry too much about the rest of the app, but if there aren't too many calls to these other functions, I'll replace them.